app/views/BookList/index.tsx

```diff
@@ -185,7 +188,9 @@ function Explore(props: Props) {
         language?: string;
     } | undefined;
 
-    // const canAddBook = user?.permissions.includes('CAN_CREATE_BOOK');
+    const canPublisherCreateBook = isDefined(user?.publisherId);
```

Comment:
Let's change the variable name to isPublisher
```suggestion
    const isPublisher = isDefined(user?.publisherId);
```